channels-rke2.yaml

```diff
@@ -615,6 +615,17 @@ releases:
         version: 0.1.1300
     featureVersions: &featureVersions-v1
       <<: *featureVersions-v1-22-11-rke2r1
+  - version: v1.22.14-rc1+rke2r1
```

Comment:
CI failure suggests that the rke2-canal chart needs to be bumped to v3.23.3-build2022081001 
* https://github.com/rancher/rke2/pull/3287

Reply:
This is done. Will confirm on other versions/branches.

Reply:
```
Checking rke2 v1.23.11-rc1+rke2r1 rancher-rke2-charts/rke2-canal@v3.22.2-build2022050902
Unexpected chart URL for rancher-rke2-charts/rke2-canal@v3.22.2-build2022050902: https://rke2-charts.rancher.io/assets/rke2-canal/rke2-canal-v3.23.3-build2022081001.tgz
```
Same bump for all the branches, it looks like